syft/util.py

```diff
@@ -0,0 +1,78 @@
+from syft import check
+import inspect
+
+
+@check.type_hints
+def is_static_method(klass: type, attr: str):
```

Comment:
According to [PEP 8](https://www.python.org/dev/peps/pep-0008/#descriptive-naming-styles), the convention for avoiding naming conflicts with reserved keywords is to use a single trailing underscore (like `class_`.)